src/gui/widgets/settings/searchsettings.ui

```diff
@@ -192,12 +192,29 @@ Changing this value from the default is not recommended.</string>
          </item>
         </layout>
        </item>
+       <item>
+        <layout class="QVBoxLayout" name="layoutNewLine">
+         <item>
+          <widget class="QCheckBox" name="checkGlossaryList">
+           <property name="enabled">
+            <bool>true</bool>
+           </property>
+           <property name="text">
+            <string>Show lines of glossary in an unordered list (restart to take effect)</string>
```

Comment:
Don't assume anyone will know what an unordered list is. There is also no reason for this to require a restart to take effect. As long as the correct code is in `initSearch` it shouldn't.

Replace this with
>Use bullet points in term glossaries